server/routes/index.ts

```diff
@@ -66,5 +75,55 @@ export default function routes(): Router {
     }
   })
 
+  get('/appointments', async (req, res, next) => {
+    const pastAppointments = [
+      {
+        date: 'Thursday 20 Jan 2025',
+        time: '10:00am',
+        title: 'Probation appointment',
+        location: '123 Garden Street, London SE1 7TH',
+        contact: 'Karen Smith',
+        contactLink: '',
+      },
+      {
+        date: 'Thursday 15 Jan 2025',
+        time: '2:00pm',
+        title: 'Probation appointment',
+        location: 'National Probation Service, 235 Greenwich High Road, London SE10 8NB',
+        contact: 'Julie Myers',
+        contactLink: '',
+      },
+    ]
+
+    const upcomingAppointments = [
```

Comment:
We need to get these out into services this week, and split our routes (or maybe use controllers) Not in this PR though